libs/image-editor/ImageEditor/src/androidTest/kotlin/org/wordpress/android/imageeditor/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.imageeditor
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.assertEquals
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
I think we might want to remove this class, wdyt?

Reply:
Removed 3b2749e.